packages/core/content-manager/admin/src/pages/EditView/components/FormLayout.tsx

```diff
@@ -23,9 +23,11 @@ export const ResponsiveGridItem = styled(Grid.Item)`
   }
 `;
 
-interface FormLayoutProps extends Pick<EditLayout, 'layout'> {}
+interface FormLayoutProps extends Pick<EditLayout, 'layout'> {
+  hasBackground?: boolean;
+}
 
-const FormLayout = ({ layout }: FormLayoutProps) => {
+const FormLayout = ({ layout, hasBackground = false }: FormLayoutProps) => {
```

Comment:
I needed the same stuff

Reply:
btw to avoid regression errors I think we need to set as default value hasBackground to true

Reply:
what we have in develop (so what the edit view uses) is the equivalent of hasBackground=false here 🤔 because on the edit view it's not rendered on top of a white background. but this shows that the name is pretty bad because it's confusing